.mergify.yml

```diff
@@ -1,10 +1,66 @@
 pull_request_rules:
-  - name: automatically merge scala-steward's PRs
+  - name: delete head branch after merge
+    conditions: []
+    actions:
+      delete_head_branch: {}
+
+  - name: request reviews and label scala-steward's PRs
+    conditions:
+      - author=scala-steward
+    actions:
+      request_reviews:
+        users: [janstenpickle, catostrophe]
+      label:
+        add: [dependency-update]
+
+  - name: automatically merge scala-steward's PRs affecting project plugins
+    conditions:
+      - author=scala-steward
+      - status-success~=test \(2.12
+      - status-success~=test \(2.13
+      - "#files=1"
+      - files=project/plugins.sbt
+    actions:
+      merge:
+        method: merge
+
+  - name: automatically merge scala-steward's PRs affecting project build properties
+    conditions:
+      - author=scala-steward
+      - status-success~=test \(2.12
+      - status-success~=test \(2.13
+      - "#files=1"
+      - files=project/build.properties
+    actions:
+      merge:
+        method: merge
+
+  - name: automatically merge scala-steward's PRs updating semver-patch versions
     conditions:
       - author=scala-steward
       - status-success~=test \(2.12
       - status-success~=test \(2.13
       - body~=labels:.*semver-patch
     actions:
       merge:
-        method: merge
\ No newline at end of file
+        method: merge
+
+  - name: automatically merge other scala-steward's PRs reviewed by 1 maintainer
+    conditions:
+      - author=scala-steward
+      - status-success~=test \(2.12
+      - status-success~=test \(2.13
+      - "#approved-reviews-by>=1"
```

Comment:
hm... or maybe "#approved-reviews-by>=2"

Reply:
I think this is fine as 1 for now, if we add more maintainers in the future let's increase it.